root/release/edit/information.tt

```diff
@@ -314,110 +314,253 @@
 </div>
 
 <div class="documentation" data-bind="with: rootField.release">
+  <div class="bubble" data-bind="bubble: $root.titleBubble">
+    <p>
+      [% l('Please see the {doc|style guidelines} for more information.',
+           { doc => { href => doc_link('Style/Titles'), target => '_blank' } }) %]
+    </p>
+  </div>
+
+  <div class="bubble" data-bind="bubble: $root.artistBubble">
+    <p>
+      [% l('Please see the {doc|documentation} and {style|style guidelines} for more information.',
```

Comment:
I'm not sure what the indenting ruses are for TT files. Hopefully what I'm doing looks okay...